node/src/blockchain/blockchain_interface/blockchain_interface_web3/mod.rs

```diff
@@ -2235,85 +2226,4 @@ mod tests {
             TRANSFER_METHOD_ID,
         );
     }
-
-    #[test]
-    fn non_clandestine_can_fetch_latest_block_number_successfully() {
```

Comment:
I saw three tests being removed during the merge, I don't know why they were removed. I doubt, it needs your attention. 

Here are the names of those tests:

- `non_clandestine_can_fetch_latest_block_number_successfully`
- `non_clandestine_can_handle_latest_null_block_number_error`
- `non_clandestine_can_handle_latest_string_block_number_error`

Reply:
Got it!